packages/amplication-server/src/core/build/build.service.spec.ts

```diff
@@ -30,9 +30,11 @@ import { BackgroundService } from '../background/background.service';
 import { WINSTON_MODULE_PROVIDER } from 'nest-winston';
 import { CreateGeneratedAppDTO } from './dto/CreateGeneratedAppDTO';
 import { EnumActionStepStatus } from '../action/dto/EnumActionStepStatus';
+import { EnumActionLogLevel } from 'amplication-data/dist/models';
 
 jest.mock('winston');
 jest.mock('amplication-data-service-generator');
+//jest.mock('semver');
```

Comment:
Please remove commented out code.
Why is this commented out though? Can we mock and test this module?